t/cli/test_main.sh

```diff
@@ -426,7 +426,7 @@ tests:
 
 make init
 
-if ! grep "env TEST_ENV=very-long-domain-with-many-symbols.absolutely-non-exists-123ss.com:1234/path?param1=value1;" conf/nginx.conf > /dev/null; then
+if ! grep "env TEST_ENV;" conf/nginx.conf > /dev/null; then
```

Comment:
this could lead to a breaking change?

Reply:
No, because when rendering nginx.conf, apisix cli decides to insert the relevant env directives based on whether they exist or are valid. The declarative method is sufficient. The final reading still needs to use `os.getenv`, `env FOO=bar; `this method is not necessary and can be replaced by the declarative method to ensure compatibility.